2_pvc_destination_gen/README.md

```diff
@@ -19,16 +43,42 @@ mig_storage_class_mappings:
   glusterfs-storage-block_RWO: ocs_storagecluster-ceph-rbd
 ```
 
-3. Run playbook while KUBECONFIG is set for connection to **destination cluster**
+3. Run `Stage 2 (a)` while KUBECONFIG is set for connection to **destination cluster**
 ```
 export KUBECONFIG="/path/to/destination_cluster_kubeconfig"
-ansible-playbook pvc-destination-gen.yml
+ansible-playbook pvc-destination-gen.yml --tags stage_a
 ```
-4. Verify that PVCs are created on destination
+Check the report generated in the output directory in file `pvc-data-adjusted.json`. Make sure that the new PV sizes are accepteable. 
+
+A sample output from `Stage 2 (a)` :
+
+```json
+[
+    {
+        "actual_pv_size": "1019M",
+        "adjusted_pvc_capacity": "2Gi",
+        "original_pvc_capacity": "1Gi",
+        "original_volume_usage": "100%",
+        "pvc_name": "pvc-2",
+        "pvc_namespace": "pvc-migrate-bmark-0",
+        "reason": "PV was identified to be almost full. pvc-migrate grew the original size by 5%."
+    }
+]
+```
+
+In the above example, the volume on the source was full. Therefore, `pvc-migrate` grew the volume and rounded the value to the nearest possible value maintaining the original size unit `Gi`. 
```

Comment:
Not that I expect we'll run into many terabyte scale PVCs, but would this also bump 1 Ti to 2Ti?

Reply:
@djwhatle According to current logic, yes, it will. Open for suggestions to better handle this case.

Reply:
Improved logic around this. According to new approach, original `G` units will be converted to `M`.